# Exception Handling

a=10
if a==10
    print("ajith")

It is of two types they were:
Syntax error
Runtime error


# With Try-Except

print("ajith")
try:
    print(10/0)
except ZeroDivisionError:
    print("saran")
print("susmitha")


print("Executing the Type-error")
print("Zybisys")
try:
    print(1+"2")
except  TypeError:
    print("Successfully Executed the Type-error")
print("done")


print("Executing the Control-flow in Try")
try:
    print(10/0)
    print("ajith")
    print(7/0)
except ZeroDivisionError:
    print("welcome")
print("bye")


# print(int("10"))
# print(int("ajith"))

# Value-error
print("executing the Value-error")


